depmap_omics_upload/tracker.py

```diff
@@ -40,22 +40,17 @@ def __init__(self, gumbo_env="production"):
         self.screen_table_index = config["screen_table_index"]
         self.str_table_name = config["str_table_name"]
         self.str_table_index = config["str_table_index"]
-        # prod env for gumbo
-        self.client = gumbo_client.Client(username=config["gumbo_client_username"])
-        if gumbo_env == "staging":
-            # hard-coded for now, waiting for independent staging envs to be enabled
-            self.client = gumbo_client.Client(
-                config_dir="~/.config/gumbo-staging",
-                username=config["gumbo_client_username"],
-            )
+        self.client = gumbo_rest_client.Client(
+            authed_session=gumbo_rest_client.create_authorized_session(
+                use_default_service_account=False
+            ),
+            username="depmap_omics_upload",
+            base_url=gumbo_rest_client.const.prod_url
+            if gumbo_env == "production"
+            else gumbo_rest_client.const.staging_url,
+        )
         self.mapping_utils = gumbo_utils.NameMappingUtils()
 
-    def commit_gumbo(self):
-        self.client.commit()
-
-    def close_gumbo_client(self):
```

Comment:
This still gets called in the `depmap_omics` repo. Is it considered redundant in the new client? Should we remove those calls in `depmap_omics`?

Reply:
Yes, I have changes in a local branch of depmap_omics for things like this I'm working on.